ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template1.kt

```diff
@@ -98,6 +98,7 @@ private fun ColumnScope.Template1MainContent(state: PaywallState.Loaded) {
                 fontWeight = FontWeight.Black,
                 textAlign = TextAlign.Center,
                 color = colors.text1,
+                textFillMaxWidth = true,
```

Comment:
All the old templates will keep this to `true`. For Paywalls V2, the default will be false.

Reply:
Will it ever be true for Paywalls V2? If so, when?

Reply:
Probably not, since I think the correct behavior is to have the markdown just use the required space. I guess it could be used if for some reason we wanted all texts in the markdown to fill the available space... But it's not likely IMO.

Reply:
Alright, I think so too. Thanks for explaining!